src/components/map/GoogleMap.svelte

```diff
@@ -107,6 +146,11 @@
 			container.style.position = 'absolute';
 			this.getPanes().overlayMouseTarget.appendChild(container);
 		};
+		overlay.onRemove = function () {
+			if (container.parentNode) {
```

Comment:
use optional chaining here too to get rid of the `if` statement

Reply:
removed the `if` statement in place of optional chaining.